.pipelines/build.yml

```diff
@@ -0,0 +1,653 @@
+      Contents: |
+        cppwinrtvisualizer.dll
+        cppwinrtvisualizer.pdb
+        cppwinrtvisualizer.vsdconfig
+      TargetFolder: $(Build.ArtifactStagingDirectory)\Component
+
+  - task: CopyFiles@2
+    displayName: Stage Standalone cppwinrtvisualizer.*
+    condition: and(or(eq(variables['BuildPlatform'], 'x86'), eq(variables['BuildPlatform'], 'x64'), eq(variables['BuildPlatform'], 'arm64')), eq(variables['BuildConfiguration'], 'release'), in(variables['Build.Reason'], 'Manual'))
+    inputs:
+      SourceFolder: $(Build.SourcesDirectory)\natvis\$(BuildPlatform)\$(BuildConfiguration)\Standalone
+      Contents: |
+        cppwinrtvisualizer.dll
+        cppwinrtvisualizer.pdb
+        cppwinrtvisualizer.vsdconfig
+      TargetFolder: $(Build.ArtifactStagingDirectory)\Standalone
+
+  - task: CopyFiles@2
+    displayName: Stage cppwinrt_fast_forwarder.lib
+    condition: and(eq(variables['BuildConfiguration'], 'release'), in(variables['Build.Reason'], 'Manual'))
+    inputs:
+      SourceFolder: $(Build.SourcesDirectory)\_build\$(BuildPlatform)\$(BuildConfiguration)
+      Contents: cppwinrt_fast_forwarder.lib
+      TargetFolder: $(Build.ArtifactStagingDirectory)
+
+  - task: ManifestGeneratorTask@0
+    displayName: 'Manifest Generator '
+    condition: and(eq(variables['BuildPlatform'], 'x86'), eq(variables['BuildConfiguration'], 'release'), in(variables['Build.Reason'], 'Manual'))
+    inputs:
+      BuildDropPath: $(Build.ArtifactStagingDirectory)\cppwinrt
+
+  - task: PublishPipelineArtifact@0
+    displayName: Publish Artifacts
+    condition: and(eq(variables['BuildConfiguration'], 'release'), in(variables['Build.Reason'], 'Manual'))
+    inputs:
+      artifactName: $(BuildConfiguration)_$(BuildPlatform)
+      targetPath: $(Build.ArtifactStagingDirectory)
+
+  - task: PublishSymbols@2
+    displayName: Publish symbols
+    condition: and(eq(variables['BuildConfiguration'], 'release'), in(variables['Build.Reason'], 'Manual'))
+    inputs:
+      SymbolsFolder: $(Build.ArtifactStagingDirectory)
+      SearchPattern: '**/*.pdb'
+      IndexSources: false
+      SymbolServerType: TeamServices
+      SymbolsProduct: CppWinRT
+
+- job: BuildInternal
+  displayName: Build Internal Packages (VPacks)
+  dependsOn: BuildBinaries
+  condition: and(succeeded(), ne(variables['SkipInternalPackages'], 'true'), eq(variables['BuildConfiguration'], 'release'), in(variables['Build.Reason'], 'Manual'))
+  pool:
+    name: Azure Pipelines
+    vmImage: 'windows-2022'
+  steps:
+  - checkout: self
+    clean: true
+    persistCredentials: True
+
+  - task: PkgESSetupBuild@12
+    displayName: Package ES - Setup Build
+    inputs:
+      branchVersionExcludeBranch: master
+      disableWorkspace: true
+      disableMsbuildVersion: true
+      disableBuildTools: true
+
+  - task: UseDotNet@2
+    displayName: 'Use .NET Core runtime 2.1.x (required for ESRP signing)'
+    inputs:
+      version: 2.1.x
```

Comment:
This step seems to be why publish vpacks is failing.  .NET 3 or less is out of support and is blocked now.

Reply:
Seems we're in a bit of a pickle. The ESRP signing task has been failing for weeks because it requires .NET Core 2.1, or at least that's what it looked like to me:

https://microsoft.visualstudio.com/Dart/_build/results?buildId=62315308&view=logs&j=33e667ca-6ac0-5edd-a669-2247a07fa473&t=53936be5-a3bb-56e6-88f6-8ad4a2fccc05

Reply:
Can use add a second `UseDotNet` task after signing and before publishing to move to a more recent version?

Reply:
Updating from EsrpCodeSigning@1 to EsrpCodeSigning@2 fixed the issue. No longer depend on .NET Core 2.1.x.